_sass/minima/_base.scss

```diff
@@ -252,3 +252,13 @@ table {
     border: 1px solid $grey-color-light;
   }
 }
+
+.matrix {
+  padding: 0 20px;
+ 
+  .matrix-table {
+    overflow-x: auto;
```

Comment:
✨This CSS makes it look much better!!

However - it doesn't really belong in the Minima base theme file.
Can you make a new file in `_sass` called `matrix.scss` and put these lines there?